src/vs/workbench/contrib/accessibility/browser/accessibilityConfiguration.ts

```diff
@@ -771,6 +771,7 @@ const configuration: IConfigurationNode = {
 					...announcementFeatureBase
 				},
 			},
+			tags: ['sound', 'tool', 'confirmation', 'audio', 'chat', 'agent', 'notification', 'alert', 'prompt', 'interaction', 'copilot', 'assistant']
```

Comment:
Tags act more as categories. I think this setting should only have the `accessibility` tag so that it can be found with `@tag:accessibility`.